### PR TITLE
Taking a screenshot with _TimeViewer

### PR DESCRIPTION
**Pull request by GuillaumeFavelier**
This PR fixes the issue with taking screenshot with `time_viewer=True` by wrapping `_Brain.screenshot()`

Fixes one item of #7526 

*Note: it is the second time I do such function wrapping, maybe using inheritance instead of composition should be considered*

### CODE REVIEW

**Comment by GuillaumeFavelier** [comment]
The following does not seem related:

```
doc/changes/0.20.inc:399: Sherif  ==> Sheriff
Makefile:111: recipe for target 'codespell-error' failed
make: *** [codespell-error] Error 1
```

**Comment by larsoner** [comment]
I'm not sure whether or not we want the interface disabled during the screenshot, as it won't match what the user sees (a key goal of sphinx-gallery capturing). In this case I'd rather just do `time_viewer=False` in the calls

File: examples/visualization/plot_publication_figure.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
 evoked.plot()
 
 stc.plot(views='lat', hemi='split', size=(800, 400), subject='sample',
-         subjects_dir=subjects_dir, initial_time=max_t)
+         subjects_dir=subjects_dir, initial_time=max_t, time_viewer=False)
```
Comment (larsoner) [inline examples/visualization/plot_publication_figure.py]: show_traces=False, too?

Comment (GuillaumeFavelier) [reply examples/visualization/plot_publication_figure.py]: normally, only `time_viewer=False` should be necessary but it's maybe clearer with `show_traces=False` too

**Comment by larsoner** [comment]
Appears to still require merge @GuillaumeFavelier

**Comment by GuillaumeFavelier** [comment]
Oh sorry I forgot to set `show_traces=False`

**Comment by larsoner** [comment]
Thanks for the quick fix @GuillaumeFavelier

**Pull request merged**